lib/cog_api/http/permissions.ex

```diff
@@ -1,15 +1,15 @@
 defmodule CogApi.HTTP.Permissions do
-  import CogApi.HTTP.Base
-
   alias CogApi.Endpoint
+  alias CogApi.HTTP.Base
   alias CogApi.Resources.Permission
 
-  def permission_index(%Endpoint{}=endpoint) do
-    get(endpoint, "permissions") |> format_response("permissions", [%Permission{}])
+  def index(%Endpoint{}=endpoint) do
+    Base.get(endpoint, "permissions")
+    |> Base.format_response("permissions", [%Permission{}])
   end
 
-  def permission_create(%Endpoint{}=endpoint, name) do
-    post(endpoint, "permissions", %{"permission" => %{"name" => name}})
-    |> format_response("permission", %Permission{})
+  def create(%Endpoint{}=endpoint, name) do
+    Base.post(endpoint, "permissions", %{"permission" => %{"name" => name}})
```

Comment:
We can't use the bare `HTTP.Base` functions as `import` now since those names conflict with the other function names so this uses the `alias` instead of the `import` to clarify that.